mifospay/src/main/res/layout/fragment_make_transfer.xml

```diff
@@ -110,15 +110,67 @@
 
         </LinearLayout>
 
-        <View
-            android:id="@+id/view_transfer_success"
-            android:layout_width="130dp"
-            android:layout_height="130dp"
-            android:layout_gravity="center"
-            android:layout_marginBottom="@dimen/value_20dp"
-            android:layout_marginTop="@dimen/value_20dp"
-            android:background="@drawable/transfer_success"
-            android:visibility="gone"/>
+        <RelativeLayout
+            android:id="@+id/layout_success"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:visibility="gone">
+
+            <View
+                android:id="@+id/view_transfer_success"
+                android:layout_width="@dimen/value_80dp"
+                android:layout_height="@dimen/value_80dp"
+                android:layout_gravity="center"
+                android:layout_marginBottom="@dimen/value_20dp"
+                android:layout_marginTop="@dimen/value_20dp"
+                android:layout_marginRight="@dimen/value_20dp"
+                android:background="@drawable/transfer_success"
+                android:layout_alignParentRight="true"
+                android:visibility="visible"/>
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/payment_success"
```

Comment:
View Id ideally should tell the use of the view, here it's not related to the its purpose